### DISCUSSION TITLE
AttributeError: 'builtins.Graph' object has no attribute 'add_node'

### DISCUSSION THREAD

**Discussion by izzy-was** [Pometry/Raphtory#1807] [Q&A] [answered]
I'm trying to get started with Raphtory and falling at the first hurdle. I can create an empty temporal graph but when I add nodes with add_node() I get 'AttributeError: 'builtins.Graph' object has no attribute 'add_node''.

Anything obvious I've missed in the installation etc?
![Screenshot 2024-10-03 152648](https://github.com/user-attachments/assets/c748c418-abd0-46cd-a2a4-513f8715a952)

**Comment by miratepuffin** [comment, marked as answer]
Hi @izzy-was,

What version of Raphtory do you have installed? It could be you have quite a old version where this function was called add_vertex.

The latest version is 0.12.1

**Reply by izzy-was** [reply]
That’s exactly it! Thank you

**Reply by miratepuffin** [reply]
Not a worry at all! Feel free to ask any other questions as you progress with the tool :) 

We are also on slack (https://join.slack.com/t/raphtory/shared_invite/zt-1998kdihw-nna1f23GH25JJ46Apn0gjA) if you want to DM some data specific questions.
